install/kubernetes/helm/istio/README.md

```diff
@@ -116,6 +116,11 @@ The chart deploys pods that consume minimum resources as specified in the resour
     EOF
     ```
 
+1. Add `istio.io` chart repository and point to the daily release:
+    ```
+    $ helm repo add istio.io https://storage.googleapis.com/istio-prerelease/daily-build/master-latest-daily/charts
+    ```
```

Comment:
I'm pretty sure this file is shipped in the release binary.  In that case, it is confusing to tell user this step as the step is only for daily build, correct?

Reply:
Yes. You are right. How about replace the url with a real release url during release? so from code, we still see it points to daily build. but in release binary, it will be release build location.

BTW, I have checked, there is no such file existing in `https://storage.googleapis.com/istio-release/releases/1.1.0-snapshot.2/charts/index.yaml`. who is responsible to create it? Thanks.

Reply:
check with @hklai on that.  I agree index.yaml would be good to have.

Reply:
+1 on replace the url during release binary build process

Reply:
index.yaml is created by the repository creation process.  @hklai or @mandarjog or both are working on a solution to provide these helm charts via repo.

Reply:
will you update this page as well?  https://preliminary.istio.io/docs/setup/kubernetes/helm-install/#option-2-install-with-helm-and-tiller-via-helm-install

Reply:
@vnugent update this page in https://github.com/istio/istio.io/pull/2876/files